spec/support/api_spec_helper.rb

```diff
@@ -93,7 +93,7 @@ def init_api_spec_env
                      policy_profiles providers provision_dialogs provision_requests rates
                      reports request_tasks requests resource_pools results roles security_groups
                      servers service_dialogs service_catalogs service_orders service_requests
-                     service_templates services settings tags tasks templates tenants users vms zones
+                     service_templates services settings tags tasks templates tenants users virtual_templates vms zones
```

Comment:
@chriskacerguis could you also update spec/requests/api/collections_spec.rb to test the primary collection for virtual_templates returns one.